node/core/pvf/src/prepare/queue.rs

```diff
@@ -76,6 +79,8 @@ struct JobData {
 	/// The priority of this job. Can be bumped.
 	priority: Priority,
 	pvf: Pvf,
+	/// The timeout for the preparation job.
+	compilation_timeout: Duration,
```

Comment:
Nit: I think here (and elsewhere) it's more correct to say `preparation`. The worker will perform preparation, which is the combination of prevalidation and compilation (production of the compiled artifact).